app/services/record_participant_event.rb

```diff
@@ -20,11 +18,19 @@ def call
 
 private
 
+  def schema_validation
+    errors = participant_event_validator.call(config, body: raw_event)
```

Comment:
Does this read the schema file on _every_ request?

Reply:
Currently, yes. That can be optimised if it's an issue though. The validator will work with an in-memory schema object.